lib/cli/cliHost.js

```diff
@@ -69,10 +69,18 @@ module.exports = class CLIHost extends CLICommand {
         /** @type {string} */
         const hostname = args[1];
         if (!hostname) {
-            CLI.error.requiredArgumentMissing('hostname', 'host remove newHostname');
+            CLI.error.requiredArgumentMissing('hostname', 'host remove hostname');
             return void callback(34);
         }
 
+        const newHostname = tools.getHostName();
+
+        if (hostname === newHostname) {
+            // avoid object deletion when target host is same host
+            CLI.error.dontRemoveCurrentHost(hostname);
+            return void callback(35);
```

Comment:
Has the number here any meaning? I guess we should prefer enums as in exitCodes.js

Reply:
@GermanBluefox ?

Reply:
It was just some number, so I could find the place where the exit code was generated.